test/webserv/integration/test_integration.py

```
@@ -1,11 +1,37 @@
+import os
+import subprocess
+
 import pytest
-from common_functions import send_request_and_assert_response
-from common_response import *
+from common import *
```

Ruff

`from common import *` used; unable to detect undefined names

(F403)